automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -0,0 +1,185 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.firecloud.config.{AuthToken, Config}
+
+import scala.concurrent.duration._
+
+
+trait Orchestration extends FireCloudClient with LazyLogging {
+
+  def responseAsList(response: String): List[Map[String, Object]] = {
+    mapper.readValue(response, classOf[List[Map[String, Object]]])
+  }
+
+  private def retry[T](remainingBackOffIntervals: Seq[FiniteDuration])(op: => () => Boolean): Boolean = {
+    op() match {
+      case true => true
+      case false if remainingBackOffIntervals.isEmpty => false
+      case false if remainingBackOffIntervals.nonEmpty =>
+        Thread sleep remainingBackOffIntervals.head.toMillis
+        retry(remainingBackOffIntervals.tail)(op)
+    }
+  }
+
+  object billing {
+
+    object BillingProjectRole extends Enumeration {
```

Comment:
Okay. I can add that. I think I may have backed away from the type aliasing because it wasn't intuitive to me how to find usages of the enum in IntelliJ. I see now that you have to find usages of the type rather than the object which I guess makes sense.